qa/acceptance/spec/shared_examples/cli/logstash-plugin/install.rb

```diff
@@ -0,0 +1,59 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+require "logstash/devutils/rake"
+
+shared_examples "logstash install" do |logstash|
+  before(:each) do
+    logstash.install(LOGSTASH_VERSION)
+  end
+
+  after :each do
+    logstash.uninstall
+  end
+
+  describe "on #{name}" do
```

Comment:
I created https://github.com/elastic/logstash/pull/5323 to address this, howerver I find this line is wrong, name is here an rspec variable, we should either use something like `logstash.hostname` as in the other PR.